Lib/concurrent/futures/process.py

```diff
@@ -855,3 +856,31 @@ def shutdown(self, wait=True, *, cancel_futures=False):
         self._executor_manager_thread_wakeup = None
 
     shutdown.__doc__ = _base.Executor.shutdown.__doc__
+
+    def terminate_workers(self, signal=signal.SIGTERM):
+        """Attempts to terminate the executor's workers using the given signal.
+        Iterates through all of the current processes and sends the given signal if
+        the process is still alive.
+
+        After terminating workers, the pool will be in a broken state and no longer usable.
+
+        Args:
+            signal: The signal to send to each worker process. Defaults to
+                signal.SIGTERM.
+        """
+        if not self._processes:
+            return
+
+        for pid, proc in self._processes.items():
+            try:
+                is_alive = proc.is_alive()
+            except ValueError:
+                # The process is already exited/closed out.
+                is_alive = False
+
+            if is_alive:
+                try:
+                    os.kill(pid, signal)
+                except ProcessLookupError:
+                    # The process just ended before our signal
+                    pass
```

Comment:
```suggestion
             try:
                 os.kill(pid, signal)
             except ProcessLookupError:
                 # The process just ended before our signal
                 pass
```

Reply:
@csm10495 sorry for this one but the indentation was wrong :') (I hate web reviews..)